rootfs/templates/wrapper

```diff
@@ -84,13 +84,29 @@ function use() {
     # Implement DNS fix related to https://github.com/docker/docker/issues/24344
     DOCKER_ARGS+=("--dns=${DOCKER_DNS}")
   fi
-
+     
```

Comment:
Let's introduce a `LOCAL_HOME` variable. The user can define it, then we use it. If they don't define it, then we do this....

e.g.

```
if [ -z "${LOCAL_HOME}" ]; then
# not defined
...
fi
```